app/src/main/java/com/nervousfish/nervousfish/activities/CreateProfileHelper.java

```diff
@@ -40,7 +40,7 @@
     }
 
     /**
-     * Generates a KeyPair based on the type selected.
+     * Generates a list with one {@link KeyPair} based on the type selected.
```

Comment:
I added the function after that to facilitate the generating of multiple keys in create contact, is it bad that that one exists?